info/v2/container.go

```diff
@@ -64,8 +64,9 @@ type ContainerSpec struct {
 	// An example of a namespace is "docker" for Docker containers.
 	Namespace string `json:"namespace,omitempty"`
 
-	// Metadata labels associated with this container.
```

Comment:
nit: Update the description. The existing description is valid for `Labels` field. Let's add a new one for `Envs` and mention that only whitelisted env vars are exposed, to avoid users filing issues that this field is always empty.